digitalpublicgoods/digit-public-finance-management.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "DIGIT Public Finance Management",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ifix.digit.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "Installation guide - https://pfm.digit.org/",
```

Comment:
I would suggest linking to these installation instructions in the [repository](https://github.com/egovernments/iFix-Dev) or adding them directly as a README file. As of right now, a developer accessing the repo would not be able to easily identify where to find these instructions.